_i18n/en/documentation/bugs-first-aid/typo-translations.md

```diff
@@ -0,0 +1,9 @@
+Did you find a spelling mistake, or something that sounds odd? It's always possible that we made a mistake in English (in which we make the app & website), or that something has not been translated correctly. Luckily these issues are often easily fixed.
+
+# Do you want to fix it yourself?
+If the issue is in any other language than English, you can update **app translations** on [Transifex](https://www.transifex.com/antennapod/antennapod/core-values/). Issue with the **website translations** can be fixed on [Weblate](https://hosted.weblate.org/projects/antennapod/). You can [find more information in our Contribute section](/contribute/translate).
+
+Is the issue with the **English texts**? Then you will need to be or get familiar with GitHub. You can send a Pull Request for [the app](https://github.com/AntennaPod/AntennaPod/) or for [the website](https://github.com/AntennaPod/antennapod.github.io).
+
+# Do you just want to inform us?
+You can create a post in the [translations section on our forum](https://forum.antennapod.org/c/translations/11). To do so you need to create an account (if you don't have one already) or simply log in via Google. Do you not want to create an account? You can send an email to post [at] forum.antennapod.org to create a public post. Any replies will arrive to your inbox.
```

Comment:
Hmm, not sure if we should add the email address here. Even if it is obfuscated a bit. On the F-Droid forum (which has their address public), there are currently about 2-3 spam threads every day that are created via email.